### DISCUSSION TITLE
Less bloat please?

### DISCUSSION THREAD

**Discussion by ghost** [Security-Onion-Solutions/securityonion#13837] [Ideas]
Going through a fresh onion install, could someone please explain why there's a 1.24 GB fonts package? No wonder the ISO is so massive.

![image](https://github.com/user-attachments/assets/c90d28cf-cbbb-4fad-9115-5ad04c396859)

Unnecessary packages are the reason behind a long install process. What ever happened to security by simplicity?

**Comment by TOoSmOotH** [comment]
The ISO is massive due to having to have ALL containers so that airgap works properly. The package you mentioned is used by Chrome for the Security Onion desktop. The manager maintains all packages needed for all node types which is why it is rsyncing the package from the ISO into /nsm/repo. 

If you are installing a non manager via iso you can add small=1 to the boot string and it will skip copying the containers and packages to the node if you are installing using something like IPMI over a slow link. 

The long install process is because it is taking a minimal OS installation and turning it into a Security Onion grid. That process requires a specific order of operations so that things will actually work. Since it seems that you have done extensive research into the install process can you provide a list of packages that you think need to be removed? I am curious how much time we save waiting for elasticsearch to configure itself and go to green when we have removed such packages. I look forward to your suggestions!

**Reply by ghost** [reply]
I was not installing the desktop version, or an airgapped install. Why then does it need to download huge font and icon packages when selected for a standalone deployment? Where is the installation option for manager vs a single node?

**Comment by dougburks** [comment]
Hello @systematize and welcome to Security Onion Discussions.

I want to make sure that you have reviewed our discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720. In particular, this section:
> Etiquette
Please be patient, courteous, and respectful. Disrespectful messages can result in being banned.

I'm not sure if you are aware, but your messages are coming across as aggressive and condescending. There are ways to ask these kinds of questions that lead to more productive discussions.

If you have further questions or comments, please consider your tone and make sure that you are being courteous and respectful.

Thank you.
